keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -271,6 +294,153 @@ downgrade.
+
+* **Are there any tests for feature enablement/disablement?**
+  Not at this time.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  As long as the validating webhook and its associated CRD is installed before
+  enabling the feature gate, the desired result will be achieved. Enabling the
+  feature gate before installing the validation hook will allow some PVCs to
+  be created with invalid data sources, and those PVCs will never get dynamically
+  provisioned.
+
+* **What specific metrics should inform a rollback?**
+  None.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  No.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+  Simply look at the data source field of the PVCs. Non-empty data sources with
+  a Kind other than snapshot and PVC indicate this feature is in use. Also the
+  existence of any VolumePopulator.populator.storage.k8s.io CRs would indicate
+  that a populator is installed and could be used.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+the health of the service?**
+  - [ ] Metrics
+    - Metric name:
+    - [Optional] Aggregation method:
+    - Components exposing the metric:
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  At a high level, this usually will be in the form of "high percentile of SLI
+  per day <= X". It's impossible to provide comprehensive guidance, but at the very
+  high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99,9% of /health requests per day finish with 200 code
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+  implementation difficulties, etc.).
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+  This feature depends on the VolumePopulator CRD being installed, and the
+  associated validating webhook for PVCs.
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+  No
```

Comment:
You are adding a new CRD, right?  So that will result in new API calls.

Reply:
Yes but that's covered by the following question.